core/src/main/scripts/importer/updateOncokbAnnotations.py

```diff
@@ -0,0 +1,379 @@
+        'inner join sample_cna_event on sample_cna_event.CNA_EVENT_ID = cna_event.CNA_EVENT_ID '+
+        'inner join genetic_profile on genetic_profile.GENETIC_PROFILE_ID = sample_cna_event.GENETIC_PROFILE_ID '+
+        'inner join cancer_study on cancer_study.CANCER_STUDY_ID = genetic_profile.CANCER_STUDY_ID '+
+        'WHERE cancer_study.CANCER_STUDY_IDENTIFIER = "'+study_id +'"')
+        for row in cursor.fetchall():
+            alteration = list(cna_alteration_types.keys())[
+                list(cna_alteration_types.values()).index(row[2])]
+            cna += [{"id": "_".join([str(row[3]), str(row[0]), str(row[4])]), "geneticProfileId": row[0], "entrezGeneId": row[1],
+                    "alteration": alteration}]
+    except MySQLdb.Error as msg:
+        print(msg, file=ERROR_FILE)
+        return None
+    return cna
+
+def get_current_sv_data(study_id, cursor):
+    """ Get structural variant data from the current study.
+        Returns an array of dictionaries, with the following keys:
+        id, geneticProfileId, entrezGeneIdA, entrezGeneIdB, and structuralVariantType
+    """
+    sv = []
+    try:
+        cursor.execute('SELECT genetic_profile.GENETIC_PROFILE_ID, '+ 'structural_variant.SITE1_ENTREZ_GENE_ID, '+
+        'structural_variant.SITE2_ENTREZ_GENE_ID, structural_variant.EVENT_INFO, ' +
+        'structural_variant.INTERNAL_ID, structural_variant.SAMPLE_ID from cbioportal.structural_variant ' +
+        'inner join genetic_profile on genetic_profile.GENETIC_PROFILE_ID = structural_variant.GENETIC_PROFILE_ID '+
+        'inner join cancer_study on cancer_study.CANCER_STUDY_ID = genetic_profile.CANCER_STUDY_ID '+
+        'WHERE cancer_study.CANCER_STUDY_IDENTIFIER = "'+study_id +'"')
+        for row in cursor.fetchall():
+            sv += [{"id": "_".join([str(row[4]), str(row[0]), str(row[5])]), "geneticProfileId": row[0], "entrezGeneIdA": row[1],
+                    "entrezGeneIdB": row[2], "structuralVariantType": row[3]}]
+    except MySQLdb.Error as msg:
+        print(msg, file=ERROR_FILE)
+        return None
+    return sv
+
+def get_reference_genome(study_id, cursor):
+    """ Get reference genome from the study """
+    ref_genome = []
+    try:
+        cursor.execute('SELECT reference_genome.NAME FROM reference_genome ' +
+        'inner join cancer_study ON cancer_study.reference_genome_id = reference_genome.reference_genome_id ' +
+        'WHERE cancer_study.CANCER_STUDY_IDENTIFIER = "'+study_id +'"')
+        for row in cursor.fetchall():
+            ref_genome += [row[0]]
+        if len(ref_genome) == 1:
+            return REFERENCE_GENOME[ref_genome[0]]
+        else:
+            raise ValueError("There is an error when retrieving the reference genome, as multiple values have been retrieved: "+ref_genome)
+    except MySQLdb.Error as msg:
+        print(msg, file=ERROR_FILE)
+
+def fetch_oncokb_mutation_annotations(mutation_data, ref_genome):
```

Comment:
In the `importOncokbMutation.py`, you have similar logics. Are they different?

Reply:
The logic is similar, but the data is pre- and post-processed in the import methods before it uses the common code in `libImportOncoKb`